### PR TITLE
Apply `i18n-tasks normalize` to locales

### PR DESCRIPTION
**Pull request by Gargron**

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -39,6 +39,30 @@ fr:
     posts: Statuts
     remote_follow: Suivre à distance
     unfollow: Ne plus suivre
+  admin:
+    settings:
+      click_to_edit: Cliquez pour éditer
+      contact_information:
+        email: Entrez une adresse courriel publique
```
Comment (wxcafe) [inline config/locales/fr.yml]: "addresse mail" is more common in french

Comment (tcitworld) [inline config/locales/fr.yml]: Didn't someone changed them all to `courriel` elsewere some time ago ?

Comment (wxcafe) [inline config/locales/fr.yml]: Ah, might be, I didn't see that. I agree we need consistency... I just think email is more common. But if everything else says courriel let's keep that I guess.

Comment (tcitworld) [inline config/locales/fr.yml]: My guess is that at some we'll have a dedicated system to translate locales (transifex, weblate, pootle, others...) so the big decisions like this one will be taken by people who know the subject there. :)

**Review by koboldunderlord — APPROVED** [review]
LGTM +1

File: config/locales/fr.yml (hunk 1 above)
Comment (Gargron) [inline config/locales/fr.yml]: This PR is solely about normalizing the order of locale keys. You can and should do it yourself by running `i18n-tasks normalize` after translating something so a PR like this wouldn't be needed.

**Pull request merged**